libevm/VM.cpp

```diff
@@ -763,6 +763,9 @@ void VM::interpretCases()
 		{
 			if (!m_schedule->haveReturnData)
 				throwBadInstruction();
+			bigint const endOfAccess{bigint(m_SP[1]) + bigint(m_SP[2])};
```

Comment:
I would avoid explicit constructor here to still pretend bigint works like regular int.
```
bigint const endOfAccess = bigint(m_SP[1]) + bigint(m_SP[2]);
```